blog-homepage/src/articles.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Bio from './bio.js'
+import bookmark from './logo.svg';
+import PropTypes from 'prop-types';
+
+function Articles({ uData, styleClass }) {
+    const getLink = (ref) => {
+        return (e) => {
+            e.preventDefault();
+            alert(ref);
+        }
+    }
+
+    return (
+        uData.map((x, i) =>
```

Comment:
When possible, I'd recommend being more specific with variable names when they're being accessed. For example, a variable name like `article` or `post` would be more specific than `x`. More of a nitpick than anything else.